src/ispd/arquivo/xml/utils/WrappedElement.java

```diff
@@ -13,6 +14,49 @@
 public class WrappedElement {
     private final Element element;
 
+    /**
+     * Construct a {@link TwoStageUniform} from this instance, with the values
+     * acquired from {@link #minimum()}, {@link #maximum()} and
+     * {@link #average()}.
+     *
+     * @return {@link TwoStageUniform} with a minimum, maximum, and average values
+     * as the respective tags in this instance, and {@code 0} for  probability.
+     */
+    public TwoStageUniform toTaskSizeNoProbability() {
+        return new TwoStageUniform(this.minimum(), this.average(), this.maximum(), 0);
```

Comment:
Potentially wrong, this makes a "two stage uniform" that only picks values from one of the intervals.

Reply:
Instances created via this method are not used to generate values, only to gather the values in its fields. Still, in the future, a better concept should be created to encapsulate this.